### ISSUE TITLE
Move gh-pages to /docs folder

### ISSUE THREAD

**Issue by Jabher** [cazala/synaptic#134]
GitHub brought the new feature few days ago:
https://github.com/blog/2228-simpler-github-pages-publishing

Now it's possible to keep documentation consistent with a code by committing it all together and keep branching actual.

Possibly we should move it.

Steps to implement:
1. move content from gh-pages to master/docs
2. press the switch in admin panel

**Comment by cazala** [comment]
@Jabher by docs you mean [the website](http://synaptic.juancazala.com/), right?

**Comment by Jabher** [comment]
Yes, it's hosted now on https://github.com/cazala/synaptic/tree/gh-pages, right?

**Comment by cazala** [comment]
yes, and that's only the built website without the source. I could move everything to `/docs` and keep the source maybe in `/docs/src` so everything is in the same master branch. cos right now the source lives only in my computer, the built website is in gh-pages, and well, the library is in master.

**Comment by Jabher** [comment]
It's always better to keep sources hosted, right :)

**Comment by cazala** [comment]
It makes sense. It's morning time in Argentina right now, I can get it done at night once I get home since the sources are on my personal computer.

**Comment by Jabher** [comment]
Cool! 
Have a nice day then

**Comment by Jabher** [comment]
any news? :)
